Rectangle.cpp

```diff
@@ -1,15 +1,30 @@
 #ifndef Rectangle_cpp
 #define Rectangle_cpp
+
 #include <iostream>
-#include "Draw.cpp"
+#include "Shape.cpp"
+#include "Area.cpp"
+
 using namespace std;
 
-class Rectangle : public Draw
+class Rectangle : public Shape
 {
 public:
     void drawShape() override
     {
         cout << "rectangle" << endl;
     }
+    int inputAndCalculateArea() override
```

Comment:
This function is performing multiple operations, as indicated by the name also.
Taking input of the dimensions of shape should not be part of this function or vice versa.